tests/outputs/a_grppub_readlist/reading_lists.yaml

```diff
@@ -15,7 +15,7 @@ getting_started_with_pdf:
   year: 2019
 test_the_lister:
   papers:
-    doi: 10.1021/nn501591g
-    text: The description of symbolic computing in Python
```

Comment:
I also modified a_grppub_readlist helper a little bit, because previously, the format here wasn't list type, and it conflicted with the new schemas.

Reply:
good!